contracts/core/funds-distribution-token/v1/BasicFDT.sol

```diff
@@ -40,7 +37,7 @@ abstract contract BasicFDT is IBaseFDT, ERC20 {
                    in a distribution can be less than 1 (base unit).
                 We can actually keep track of the undistributed funds in a distribution
                    and try to distribute it in the next distribution.
-    */
+     */
     function _distributeFunds(uint256 value) internal {
```

Comment:
Does the NatSpec for internal functions get generated in the docgen?

Reply:
No.